[2025-10-26T19:14:21.660+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-26T19:14:21.688+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Dag_OpenFlightsDataIngestion.group_routes.create_table_routes manual__2025-10-26T19:14:08.334532+00:00 [queued]>
[2025-10-26T19:14:21.702+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Dag_OpenFlightsDataIngestion.group_routes.create_table_routes manual__2025-10-26T19:14:08.334532+00:00 [queued]>
[2025-10-26T19:14:21.702+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-10-26T19:14:21.721+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): group_routes.create_table_routes> on 2025-10-26 19:14:08.334532+00:00
[2025-10-26T19:14:21.731+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=516) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-26T19:14:21.734+0000] {standard_task_runner.py:63} INFO - Started process 573 to run task
[2025-10-26T19:14:21.734+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Dag_OpenFlightsDataIngestion', 'group_routes.create_table_routes', 'manual__2025-10-26T19:14:08.334532+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/openflights_dag.py', '--cfg-path', '/tmp/tmpwyb1np14']
[2025-10-26T19:14:21.737+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask group_routes.create_table_routes
[2025-10-26T19:14:21.814+0000] {task_command.py:426} INFO - Running <TaskInstance: Dag_OpenFlightsDataIngestion.group_routes.create_table_routes manual__2025-10-26T19:14:08.334532+00:00 [running]> on host c8d58475e5b6
[2025-10-26T19:14:21.942+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Dag_OpenFlightsDataIngestion' AIRFLOW_CTX_TASK_ID='group_routes.create_table_routes' AIRFLOW_CTX_EXECUTION_DATE='2025-10-26T19:14:08.334532+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-26T19:14:08.334532+00:00'
[2025-10-26T19:14:21.945+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-26T19:14:21.960+0000] {base.py:84} INFO - Using connection ID 'maria_db_default' for task execution.
[2025-10-26T19:14:21.961+0000] {mariadb.py:94} INFO - Connecting to MariaDB database: 
[2025-10-26T19:14:22.247+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-26T19:14:22.248+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-26T19:14:22.262+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Dag_OpenFlightsDataIngestion, task_id=group_routes.create_table_routes, execution_date=20251026T191408, start_date=20251026T191421, end_date=20251026T191422
[2025-10-26T19:14:22.288+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-26T19:14:22.331+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-26T19:14:22.333+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
